RELEASE-NOTES.txt

```diff
@@ -1,5 +1,9 @@
 *** PLEASE FOLLOW THIS FORMAT: [<priority indicator, more stars = higher priority>] <description> [<PR URL>]
 
+11.1
+-----
+
+
 11.0
 -----
 
```

Comment:
I was surprised by the empty notes here, but the `release_notes.txt` copy below is consistent with this 👍